README.md

```diff
@@ -40,7 +40,12 @@ To open DICAT, simply double click on the executable.
 
 To install DICAT source code on a computer, download and save the content of the current Github repository into a workstation.
 
-Before running DICAT, make sure your systems contains a [Python](https://www.python.org) compiler with the [TkInter](https://wiki.python.org/moin/TkInter) library (usually, TkInter comes by default with most Python installations. The [PyDICOM](http://www.pydicom.org) package is also required by DICAT. 
+Before running DICAT, make sure your systems contains a [Python](https://www.python.org) compiler with the [TkInter](https://wiki.python.org/moin/TkInter) library (usually, TkInter comes by default with most Python installations.
+
+For Ubuntu:

```

Comment:
Here I would say:
"For Ubuntu distributions, TkInter can be installed via apt-get:"